README.md

```diff
@@ -1 +1,3 @@
-# review-react-project
\ No newline at end of file
+# To Do React
+
+
```

Comment:
- [ ] It is a requirement that you create a descriptive ReadMe file using the provided [template](https://github.com/microverseinc/readme-template/blob/master/README.md). Kindly do so and please make sure all the sections in the readme file are filled out.